app/services/document_unpublishing_service.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class DocumentUnpublishingService
+  def retire(document, explanatory_note)
+    GdsApi.publishing_api_v2.unpublish(document.content_id, type: "withdrawal", explanation: explanatory_note)
```

Comment:
For these api calls I think it'd make sense to include the locale in this. I know for now we'll only being dealing with "en" but be good to not have surprises later when this becomes more pertinent.

Reply:
I'll update and fix up.